src/o365/pa/commands/pcf/pcf-init.ts

```diff
@@ -0,0 +1,206 @@
+import * as fs from "fs";
+import * as path from 'path';
+const uuidv4 = require('uuid/v4');
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import Command, {
+  CommandOption,
+  CommandValidate,
+  CommandAction
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import TemplateInstantiator from "./pcf-init/template-instantiator";
+import { TemplateVariables } from "./pcf-init/template-variables";
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  namespace: string;
+  name: string;
+  template: string;
+}
+
+/*
+ * Logic extracted from bolt.module.pcf.dll
+ * Version: 0.4.3
+ * Class: bolt.module.pcf.PcfInitVerb
+ */
+class PaPcfInitCommand extends Command {
+  public get name(): string {
+    return commands.PCF_INIT;
+  }
+
+  public get description(): string {
+    return 'Initializes a directory with a new PowerApps component framework project';
```

Comment:
Doesn't match the description from the spec: `Creates new PowerApps component framework project`

Reply:
True, this is the description coming from the newer version of the official PowerApps CLI.